File: BEGINNER/Java/1012-Area.java
import java.util.Scanner;

public class Main{
    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);
        
        double A = sc.nextDouble();
        double B = sc.nextDouble();
        double C = sc.nextDouble();
        
        double triangleArea = (A*C)/2;
        double circleArea = 3.14159*Math.pow(C,2);
        double trapeziumArea = (A+B)*(C/2);
        double squareArea = B*B;
        double rectangleArea = A*B;
        
        System.out.println("TRIANGULO: " + String.format("%.3f", triangleArea));
        System.out.println("CIRCULO: " + String.format("%.3f", circleArea));
        System.out.println("TRAPEZIO: " + String.format("%.3f", trapeziumArea));
        System.out.println("QUADRADO: " + String.format("%.3f", squareArea));
        System.out.println("RETANGULO: " + String.format("%.3f", rectangleArea));
    }
}

File: BEGINNER/Python/1012-Area.py
A, B, C = map(float,input().split())

triangleArea = (A*C)/2
circleArea = 3.14159*pow(C,2)
trapeziumArea = (A+B)*(C/2)
squareArea = B*B 
rectangleArea = A*B

print("TRIANGULO: %.3f" %triangleArea)
print("CIRCULO: %.3f" %circleArea)
print("TRAPEZIO: %.3f" %trapeziumArea)
print("QUADRADO: %.3f" %squareArea)
print("RETANGULO: %.3f" %rectangleArea)
